packages/awslint/lib/rules/construct.ts

```diff
@@ -93,9 +96,15 @@ constructLinter.add({
 
     const expectedParams = new Array<MethodSignatureParameterExpectation>();
 
+    let baseType;
+    if (process.env.AWSLINT_BASE_CONSTRUCT && !initializer.parentType.name.startsWith('Cfn')) {
```

Comment:
I guess that's also temporary... let's make sure to create an issue for this or at least add an item in our plan

Reply:
In the plan now. I was awaiting approval.